### ISSUE TITLE
[OccTax] Limiter la liste des observateurs pour un compte

### ISSUE THREAD

**Issue by sig-pnrnm** [PnX-SI/GeoNature#492]
Question du vendredi soir :smiley: 

_(j'aurais certainement trouvé la réponse en m'appropriant un peu mieux le modèle de données de la V2, et en particulier le CRUVED, mais c'est quand même plus rapide de poser la question. J'autorise une simple réponse par `OUI` ou `NON` !)_

Est-il possible, dans OccTax, de limiter la liste des observateurs possibles pour un utilisateur ?
En l'occurence, pour certains types d'utilisateurs, on souhaiterait qu'ils ne puissent saisir qu'en leur nom, et pas au nom des autres.

Est-ce que c'est déjà possible ?
(si la réponse est `OUI` je chercherais !)

**Comment by sig-pnrnm** [comment]
Autre exemple d'utilisation quand même : limiter la liste des observateurs aux observateurs de SA structure

**Comment by camillemonchicourt** [comment]
Non pas prévu

**Comment by Idaline** [comment]
Bonjour !

Je remets d'actualité ce billet car nous aurions une préférence pour cette option également !

**Comment by camillemonchicourt** [comment]
Pertinent en effet.
On pourrait imaginer que si un utilisateur a un CRUVED C de 1 sur Occtax, il ne peut renseigner que lui-même dans les observateurs, si il a un CRUVED de 2 que les utilisateurs de sa structure, si 3, on lui liste tous les observateurs.

**Comment by DonovanMaillard** [comment]
Ca serait top oui ! 

Le lien que ca induirait entre la liste des observateurs dispos et les jdd accessible pose un peu question (je n'ai accès qu'à mon jeu de données donc je peux pas saisir les observations avec mon copain de terrain) cependant. Mais s'il faut commencer à décorréler les 2......

**Comment by jbrieuclp** [comment]
Ce serait une chose à rendre paramétrable, parceque dans notre cas ça ne me semble pas vraiment adapté. Il y a de nombreuse sorties qui sont réalisées avec des personnes qui n'appartiennent pas à la structure. Les observateurs sont inscrit dans usershub, mais sans création de compte et sans non plus être rattaché à notre structure (ils ne sont pas forcément adhérents). Et le cas plus simple des inventaires réalisés par des membres de structures différentes ensemble : il ne sera plus possible de le saisir.

En fait cette demande me donne envie de remettre sur la table le sujet de pouvoir associer plusieurs organismes à un même compte utilisateur !

**Comment by camillemonchicourt** [comment]
Une autre possibilité est de définir un objet "observateur" dans les objets de permissions du module Occtax et de pouvoir lui associer une portée 1, 2 ou 3.

Dans ce cas on gererait le contenu de cette liste indépendamment de celle des JDD et on aurait les 3 possibilités sans inclure de nouveau paramètre.

**Comment by DonovanMaillard** [comment]
Oui, sur le fond ca serait le mieux, sur la forme je sais pas comment ca peut se traduire. Traiter les observateurs comme si c'était un module ? Comme si c'était une action ? Un nouvel objet d'un nouveau type ?

**Comment by camillemonchicourt** [comment]
Oui à priori plutôt un nouvel objet "observateurs".
